Directory.Build.targets

```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
   <ItemGroup>
-    <PackageReference Update="System.Runtime.CompilerServices.Unsafe" Version="4.6.0" />
+    <PackageReference Update="System.Runtime.CompilerServices.Unsafe" Version="5.0.0" />
     <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.3" />
-    <PackageReference Update="System.Memory" Version="4.5.3" />
+    <PackageReference Update="System.Memory" Version="4.5.4" />
```

Comment:
Shouldn't libraries require the lowest version possible, in terms of api support, so that any app using npgsql is not forced to use a higher version. Let's say 4.5.0 in this case?

Reply:
I don't know... In general we update dependencies to the highest version at the time of release, to have all possible bugfixes, perf improvements - that seems pretty standard... If users really do want to hang back with a version, they can override this and suppress the warning, no?